Tests/Engine/ModuleDependencyHandler.tests.ps1

```diff
@@ -27,15 +27,19 @@ Describe "Resolve DSC Resource Dependency" {
     }
     AfterAll {
         if ( $skipTest ) { return }
-        $env:PSModulePath = $SavedPSModulePath
+        $env:PSModulePath = $savedPSModulePath
     }
 
     Context "Module handler class" {
         BeforeAll {
             if ( $skipTest ) { return }
             $moduleHandlerType = [Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.ModuleDependencyHandler]
             $oldEnvVars = Get-Item Env:\* | Sort-Object -Property Key
-            $oldPSModulePath = $env:PSModulePath
+            $savedPSModulePath = $env:PSModulePath
+        }
+        AfterAll {
```

Comment:
Just a note, there is a Pester issue [here](https://github.com/pester/Pester/issues/300) whereby using `BeforeAll` or `AfterAll` inside a `Context` block results in unpredictable behaviour. Technically they should be only under a `Describe` block. We should address this in another PR as refactoring this test suite is hard.

Reply:
yah - I agree - the missing after all was _the_ reason that the analyzer needed to be installed into a specific location. with this change, that is no longer needed!